os_net_config/sriov_config.py

```diff
@@ -821,18 +822,17 @@ def parse_opts(argv):
     return opts
 
 
-def main(argv=sys.argv, main_logger=None):
+def main(argv=sys.argv):
     opts = parse_opts(argv)
-    if not main_logger:
-        main_logger = common.configure_logger(log_file=True)
-    common.logger_level(main_logger, opts.verbose, opts.debug)
+    logger = common.configure_logger(log_file=True)
```

Comment:
Will this overwrite the module level logging (line 38) from sriov_config -> os-net-config ?

Reply:
Yes it will